saw-core/src/Verifier/SAW/Module.hs

```diff
@@ -152,7 +152,7 @@ data Ctor =
     -- where the @ps@ are the parameters and the @ix@s are the indices of
     -- datatype @d@
   , ctorIotaReduction ::
-       Term   {- ^ recursor term -} ->
+       Term -> {- ^ recursor term -}
```

Comment:
You'll actually want to put the arrow _after_ the comment, as Haddock will fail to parse this otherwise.

Reply:
Resolved in 52eca30dfa7ffdf7760e4f70eafb8403351fbac3